frontend/src/app/testResults/TestResultInputForm.jsx

```diff
@@ -32,7 +32,7 @@ const TestResultInputForm = ({ testResultValue, onSubmit, onChange }) => {
         <Button
           onClick={onSubmit}
           type="submit"
-          disabled={testResultValue === null || testResultValue === undefined}
+          disabled={!testResultValue}
```

Comment:
Ooh, exciting. Will putting typescript on this file (if and when) make it easier to track down, or just force you to track it down as part of the process?

Reply:
Typescripting everything so that it matches the graphql queries should protect us some. I think we'd need to have specified this as an enumeration for it to be caught, because an empty string is still a `String`.